src/components/Contacts/Contacts.module.css

```diff
@@ -0,0 +1,8 @@
+.wrap {
+	width: 600px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/TODO-Olga/pullRequest?prid=5555083)